src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/rpa/RequestForJudgementNotificationServiceTest.java

```diff
@@ -140,11 +119,11 @@ public void shouldNotSendRoboticsEmailWhenCCJByAdmission() {
 
         service.notifyRobotics(event);
 
-        verifyNoInteractions(emailService);
+        verifyEmailSent();
     }
 
     @Test
-    public void shouldNotSendRoboticsEmailWhenCCJByDetermination() {
+    public void shouldSendRoboticsEmailWhenCCJByDetermination() {
```

Comment:
If we originally did not send emails for determination, and determination is out of scope for this story, I would say we should still not be sending emails in this case.